gui_handlers.go

```diff
@@ -4,6 +4,8 @@
 package main
 
 import (
+	"log"
```

Comment:
This is because if there's an error on exit, it should be dumped to STDOUT and not captured by the usual logger, where it would never be seen.